test/functional/tests/security/fuzzy/kernel/fuzzy_with_io/fuzzy_statistics/test_fuzzy_print_statistics_io_class_id_for_cache.py

```diff
@@ -26,48 +37,63 @@
 @pytest.mark.parametrizex("cleaning_policy", CleaningPolicy)
 @pytest.mark.parametrizex("unaligned_io", UnalignedIo)
 @pytest.mark.parametrizex("use_io_scheduler", UseIoScheduler)
-def test_fuzzy_print_statistics_io_class_id_for_cache(cache_mode, cache_line_size, cleaning_policy,
-                                                      unaligned_io, use_io_scheduler):
+def test_fuzzy_print_statistics_io_class_id_for_cache(
```

Comment:
meh, this one is the worst so far :D IO class id is the fuzzed parameter, so I think it should be at the end  

Applies to following tests